packages/runtime-core/__tests__/rendererChildren.spec.ts

```diff
@@ -422,10 +422,24 @@ describe('renderer: keyed children', () => {
     expect(serialize(elm.children[0])).toBe(`<div>four</div>`)
   })
 
-  test('should warn with duplicate keys', () => {
+  it('should warn with duplicate keys: mountChildren', () => {
```

Comment:
```suggestion
  test('should warn with duplicate keys: mountChildren', () => {
```

just to make it consistent with the file even though it's not consistent across the project

Reply:
Done~